docs/release-notes/2.26.0.md

```diff
@@ -0,0 +1,89 @@
+ 
+
+!!! caution alert alert-warning "Important"
+    These experimental dashboards are subject to change. It is recommended to use these dashboards for testing purposes only.   
+
+
+- **SMTP Configuration Verification**
+
+When configuring an SMTP email server for sending out email notifications, you now have the option to test that your specified settings are correct.
+The **Email* tab under **Configuration > Settings > Communication** now includes a **Test** button to send a test alert through the specified server.
+For more information about setting up an SMTP email server, see [Configure](https://www.percona.com/doc/percona-monitoring-and-management/2.x/how-to/configure.html).
+
+- **Breaking change for Integrated Alerting (Technical Preview)**
+ 
+This release introduces major changes to the core structure of rule templates. As a result, alert rules and templates created in previous PMM versions are not compatible with PMM 2.26.0 and cannot be migrated to this new version. After upgrading, make sure to manually recreate any custom alert rules and rule templates that you want to transfer to PMM 2.26.0.
+ 
+!!! caution alert alert-warning "Disclaimer"
+    Integrated Alerting is still a Preview functionality and, as such, subject to change. We recommend that early adopters use this feature for testing purposes only.
+
+
+## New Features
+
+* [PMM-9059](https://jira.percona.com/browse/PMM-9059): Following the recently introduced support for connecting PMM to Percona Platform, you now have the option to also unlink any servers that are no longer relevant to your Platform organization.
+ To disconnect a PMM server, go to **Configuration > Settings > Percona Platform** and click **Disconnect**.
+ You can check the list of servers connected to an organization in Percona Platform by clicking View instances on the Dashboard page. For more information, see the [Configure](https://www.percona.com/doc/percona-monitoring-and-management/2.x/how-to/configure.html) for more details.
+* [PMM-9312](https://jira.percona.com/browse/PMM-9312): Tech Preview Feature: PMM now captures the MongoDB metrics such as dbStats, collStats, indexStats, and topmetrics. See [Documentation](https://www.percona.com/doc/percona-monitoring-and-management/2.x/details/commands/pmm-admin.html) for more details.
+
+
+
+## Improvements
+
+* [PMM-9176](https://jira.percona.com/browse/PMM-9176): DBaaS: PMM now supports Percona Distribution for MongoDB Operator 1.10.0.
+* [PMM-9159](https://jira.percona.com/browse/PMM-9159): DBaaS: PMM now supports Percona Kubernetes Operator for Percona XtraDB Cluster 1.10.0.
+* [PMM-9180](https://jira.percona.com/browse/PMM-9180): Integrated Alerting:  Add Alert Rule: Added the Template expression in a collapsible panel for an enhanced user experience (default view as collapsed) as the technical message could confuse the users.
```

Comment:
We should avoid content in parentheses as much as possible. Does it make sense to move the info to a separate sentence instead?